LayoutTests/http/wpt/fetch/resources/status-without-message.asis

```diff
@@ -1 +1 @@
-HTTP/1.1 200
+HTTP/1.1 200 
```

Comment:
Did the old test fail in Firefox and Chrome?

Reply:
We decided to change the h1 parser, so I'll revert this test change. This is not the only test that checks this behavior, and Chrome accepts it.